provisioning/windows_imaging.md

```diff
@@ -0,0 +1,151 @@
+# Infrastructure Prerequisites and Configuration
+## Infrastructure setup:
+* Configure iPXE/sources/windows directory to be accessible via SAMBA (Create directories if they don't already exist)
+```
+      [sources]
+      path = /srv/httpboot/ipxe/sources
+      guest ok = yes
+      writable = yes
+      browsable = yes
+      printable = no
+```
+* Configure iPXE/sysprep directory to be accessible via SAMBA (Create directories if they don't already exist)
+```
+      [sysprep]
+      path = /srv/httpboot/ipxe/sysprep
+      guest ok = yes
+      writable = no
+      browsable = no
+      printable = no
+```
+* Configure iPXE menu item and get sources for WinPE
+  * Get Memdisk
+    * Obtain Syslinux package http://www.kernel.org/pub/linux/utils/boot/syslinux/
+    * Extract package
+    * get ‘memdisk’ from syslinux-v.vv/memdisk/
+  * Prepare iPXE structure TODO
+    * Log into PXE server and cd to iPXE directory
+    * Copy memdisk file into sources directory
+  * Create iPXE menu item for WinPE
+    * Menu section:
+    ```item winpex64   WindowsPE_amd64```
+    * Label:
+```
+          :winpex64
+          initrd http://server/ipxe/sources/windows/winpe_amd64.iso
+          kernel http://server/ipxe/sources/memdisk iso raw
+          boot
+```
+
+## Admin Workstation prep:
+1. Get Windows AIK and install on admin workstation http://www.microsoft.com/en-us/download/details.aspx?id=5753
+1. Build Windows_PE environment and create ISO
+  1. Generate WinPE directory structure
+    * Open Start / Windows AIK / Deployment Tools Command Prompt
+    ```copycmd amd64 c:\winpe_amd64```
+  1. Copy additional tools and customize our working directory
+    * Always enter WinPE (don't prompt for key press)
+    ```del c:\winpe_amd64\ISO\boot\bootfix.bin```
+  1. Edit the Boot Image
+    * Mount the WIM file
+    ```dism /mount-wim /wimfile:c:\winpe_amd64\winpe.wim /index:1 /mountdir:c:\winpe_amd64\mount```
```

Comment:
Specifically, the bulleted line is immediately followed by the snippet.

See below:


![screen shot 2015-02-13 at 10 56 03 am](https://cloud.githubusercontent.com/assets/19339/6190320/049f3096-b36f-11e4-8ff6-061db1699625.png)